src/daft-core/src/utils/display_table.rs

```diff
@@ -99,8 +99,8 @@ pub fn make_schema_vertical_table<F: AsRef<Field>>(fields: &[F]) -> comfy_table:
     }
 
     let header = vec![
-        comfy_table::Cell::new("Column Name").add_attribute(comfy_table::Attribute::Bold),
```

Comment:
Had to remove the bolds here because we can't put bold characters in docstrings.

Reply:
Let's set up an environment variable for this! So we can set `DAFT_BOLD_COLUMN_NAME_REPR=0` in the `env`